hello_jsontype.py

```diff
@@ -0,0 +1,57 @@
+from metaflow import FlowSpec, step, Parameter, JSONType
+
+
+class HelloJSONType(FlowSpec):
+    """
+    A flow for testing ability to handle JSONType objects.
+
+    """
+
+    str_to_list = Parameter("str_to_list",
+                            type=JSONType,
+                            default="[1, 2, 3]")
+    str_to_dict = Parameter("str_to_dict",
+                            type=JSONType,
+                            default='{"one": 1, "two": 2, "three": 3}')
+    list_param = Parameter("list_param",
+                           type=JSONType,
+                           default="[1, 2, 3]")
+    dict_param = Parameter("dict_param",
+                           type=JSONType,
+                           default='{"one": 1, "two": 2, "nine": 9}')
```

Comment:
Nice that you are testing all the cases.

Reply:
Thanks